recipes/stb/all/conanfile.py

```diff
@@ -11,46 +14,54 @@ class StbConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/nothings/stb"
     license = ("Unlicense", "MIT")
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     options = {
-        "with_deprecated": [True, False]
+        "with_deprecated": [True, False],
     }
 
     default_options = {
-        "with_deprecated": True
+        "with_deprecated": True,
     }
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _version(self):
         # HACK: Used to circumvent the incompatibility
         #       of the format cci.YYYYMMDD in tools.Version
```

Comment:
i actually learn recently that the new Version tool should actually work with it, have you tried that?

Reply:
Indeed it works:

```
>>> from conan.tools.scm import Version
>>> Version("cci.20201022") < "cci.20201122"
True
>>> Version("cci.20201022") < "cci.20200922"
False
```